integration_tests/androidx_test/src/sharedTest/java/org/robolectric/integrationtests/axt/EspressoTest.java

```diff
@@ -193,7 +193,11 @@ public void textViewWithLetterSpacing() {
 
   @Test
   public void customActivityLabel() {
-    onView(withText("Activity Label")).check(matches(isCompletelyDisplayed()));
+    activityRule.getScenario().close();
```

Comment:
Doesn't the `ActivityScenarioRule` closes the scenario by itself after each test?

Reply:
Oh right. Wouldn't it be better to split the class in two, to make it clear which test uses which `Activity`?
This would also avoid potential issues where someone forgets to close a scenario